base_export_manager/models/ir_exports.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class IrExports(models.Model):
+    _inherit = 'ir.exports'
+
+    name = fields.Char(required=True)
+    resource = fields.Char(
+        required=True,
+        readonly=True,
+        help="Model's technical name.")
+    model_id = fields.Many2one(
+        "ir.model",
+        "Model",
+        required=True,
+        store=True,
+        domain=[("osv_memory", "=", False)],
+        compute="_compute_model_id",
+        inverse="_inverse_model_id",
+        help="Database model to export.")
+
+    @api.multi
+    @api.depends("resource")
+    def _compute_model_id(self):
+        """Get the model from the resource."""
+        for s in self:
+            s.model_id = self._get_model_id(s.resource)
+
+    @api.multi
+    def _inverse_model_id(self):
+        """Get the resource from the model."""
+        for s in self:
+            s.resource = s.model_id.model
```

Comment:
You can see this method is not being called in the runbot when you select a `model_id`. I'd thank help on why.